web/src/components/AppBanner.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <v-banner
+    app
+    color="blue"
+    dark
+    icon="mdi-information"
+    class="ma-4"
+  >
+    <span class="title">Anouncement: </span>The NMDC will be replacing legacy identifiers with
+    <a href="https://microbiomedata.github.io/nmdc-schema/identifiers/#ids-minted-for-use-within-nmdc">
+      NMDC persistent identifiers
+    </a> on April 22, 2024 for studies,
+    samples, analysis workflows, and files. Additionally, annotation will be reprocessed for legacy metagenomes
+    and metatranscriptomes, along with binning for metagenomes. Legacy metaproteomics datasets will be reprocessed
+    using the new metagenome annotations. For information on mapping between old and new identifiers please refer
+    to
+    <a href="https://microbiomedata.github.io/nmdc-schema/identifiers/#reuse-vs-minting-new-ids">
+      this documentation
```

Comment:
When rendered, this link ends with an underlined space.

<img width="182" alt="image" src="https://github.com/microbiomedata/nmdc-server/assets/134325062/f9cfa6ca-d2b2-475e-a612-64cd6c5d1a6b">

Reply:
Fixed in https://github.com/microbiomedata/nmdc-server/pull/1216